src/router/index.js

```diff
@@ -84,7 +84,68 @@ const routes = [
     name:'Vendorprofileview',
     component:()=> import('@/views/user/vendor_profile_view.vue'),
     // props:(route)=>({id: route.params.id})
-  }
+  },
+  {
+    path:'/vendor_profile',
+    name:'VendorProfile',
+    component:()=> import('@/views/event_merchant/vendor_profile.vue'),
+    // props:(route)=>({id: route.params.id})
+  },
+  {
+    path:'/edit_profile',
+    name:'EditProfile',
+    component:()=> import('@/views/event_merchant/edit_profile.vue'),
+    // props:(route)=>({id: route.params.id})
+  },
+  {
+    path:'/new_service_1',
+    name:'NewService1',
```

Comment:
Can this be named better, e.g NewService[What the Page does]

Reply:
okay I will amend that sir.